test/integration/script-loader/test/index.test.js

```diff
@@ -127,6 +127,13 @@ describe('Script Loader', () => {
     let browser
     try {
       browser = await webdriver(appPort, '/')
+      await waitFor(1000)
```

Comment:
What is this waiting for? We wait for hydration to occur already before resolving the `webdriver()` promise so we can remove this if that's all it's waiting for. If it's waiting for something else can we use `check` from `next-test-utils` to watch for it instead?